spec/turn_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rspec'
+require './lib/turn'
+require './lib/card'
+
+RSpec.describe Turn do
+  it 'exists' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn_1 = Turn.new("Juneau", card_1)
+
+    expect(turn_1).to be_instance_of(Turn)
+  end
+
+  it 'returns the guess' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn_1 = Turn.new("Juneau", card_1)
+
+    expect(turn_1.guess). to eq("Juneau")
+  end 
+
+  it 'returns the card' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn_1 = Turn.new("Juneau", card_1)
+
+    expect(turn_1.card). to eq(card_1)
+  end
+
+  it 'returns true when answer is correct' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn_1 = Turn.new("Juneau", card_1)
+
+    expect(turn_1.correct?).to be true 
+  end
+
+  it 'provides Correct! feedback' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn_1 = Turn.new("Juneau", card_1)
+
+    expect(turn_1.feedback). to eq("Correct!")
+  end
+
+  it 'provides Incorrect feedback' do
```

Comment:
This is great. It's good to always test every outcome, not just the "positive" or correct outcome. I might also add a line to explicitly test the `correct?` method as well (even though it is implicitly tested here). Generally every method we write should have its own test.